docs/modules/ROOT/pages/api/account.adoc

```diff
@@ -272,6 +272,14 @@ use openzeppelin::presets::Account;
 
 Basic account contract leveraging xref:#AccountComponent[AccountComponent].
 
+:class-hashes: xref:presets.adoc[Compiled class hash]
```

Comment:
```suggestion
:class-hashes: xref:presets.adoc[Class hash]
```
why "compiled"?

Reply:
Because there's a class hash for Sierra classes as well, and we are referring to the Casm one, which is called compiled class to differentiate. We can use class hash and experienced users will understand, but new users may get confused with the Sierra class hash.

Reply:
`Class hash (CASM)` is another option